_posts/2019-05-16-released-v6.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Released Chainer/CuPy v6.0.0
+layout: post
+categories: Announcement
+author: Seiya Tokui
+---
+
+We have released Chainer and CuPy v6.0.0 today!
+This is a major release that introduces several new features.
+Full updates can be found in the release notes: [Chainer](https://github.com/chainer/chainer/releases/tag/v6.0.0), [CuPy](https://github.com/cupy/cupy/releases/tag/v6.0.0).
+
+## ChainerX
+
+The biggest update is the introduction of **ChainerX**.
+It is a fast and portable ndarray engine with autograd support written in C++ with a very thin Python wrapper.
+
+We have released the beta version of ChainerX in v6.0.0b1 as we wrote in the [previous blog post](https://chainer.org/announcement/2018/12/03/chainerx.html).
+Since then, we have been working on improving it in various aspects.
+In particular, ChainerX in v6.0.0 expands the coverage of various features since v6.0.0b1.
+
+- **Wider op coverage**.
+  We have more Chainer functions that directly call ChainerX's low-overhead implementation.
+  The effort is still on going at [the tracking issue](https://github.com/chainer/chainer/issues/6423) with [the spreadsheet of op-wise implementation status](https://docs.google.com/spreadsheets/d/1B4E78tw9Awgpcdn5G7zsQ8NVFYJdOoJlIQg42QxKNfU).
+  We continue to expand the op coverage towards the next v7 release.
+  Contributions are always welcomed!
+- **More Function coverage**.
+  Most users will start using ChainerX through Chainer's existing interface (just by replacing NumPy/CuPy arrays with ChainerX arrays).
+  When ChainerX does not have an implementation for an operation, Chainer automatically falls back to NumPy/CuPy-based implementation.
+  It basically works without any fix for most functions, but sometimes not.
+  We are fixing such bugs to enlarge the coverage of functions for ChainerX usage.
+  The effort is accompanied by the introduction of a test fixture class for function tests  (you can find [the tracking issue](https://github.com/chainer/chainer/issues/6071)).
+  The current test coverage is over 40%, mostly basic array operations resembling routines in NumPy and operations commonly used in convolutional neural networks such as convolution, deconvolution and pooling. Operations for recurrent neural networks will be addressed in the upcoming releases.
```

Comment:
`test coverage` is unclear/confusing for me because https://coveralls.io/github/chainer/chainer says 86% as of 2019/05/14 21:36.
Is there some test to measure how much ChainerX covers features/operations implemented in Chainer?

Reply:
Ah, I meant for this number that 40% of the functions under `chainer.functions` are already tested with ChainerX array inputs. I will consider rephrasing the sentence to make this point clearer.

Reply:
I modified the sentences.